### ISSUE TITLE
Better install instruction

### ISSUE THREAD

**Issue by linonetwo** [sobjornstad/TiddlyRemember#38]
I think I need to create a folder `sobjornstad` in the plugins folder first, then download zip of this repo, then extract it, and drag `tw-plugin` folder into `sobjornstad` folder, and rename `tw-plugin` to `TiddlyRemember`

<img width="431" alt="截屏2021-03-22 下午1 01 46" src="https://user-images.githubusercontent.com/3746270/111942688-c3759c80-8b0e-11eb-80ba-5cb5cd353c82.png">

**Comment by linonetwo** [comment]
Current instruction in the website is not friendly enough for new tw users.

**Comment by sobjornstad** [comment]
Hi @linonetwo,

It sounds like maybe you didn't find the installation instructions on the [front page](https://sobjornstad.github.io/TiddlyRemember/) of the TR documentation? While you can install to a folder wiki by extracting the source like you are doing, that is not required – the simplest way (and recommended for anyone who doesn't already understand how to do that) is to simply drag and drop the provided link into your wiki.

See the "Installing TiddlyRemember" tab in the bottom half of this screenshot:

![install](https://user-images.githubusercontent.com/2313414/111984441-5c89c080-8ad9-11eb-9ca2-4a26a36b1829.png)

If you were following these instructions, please elaborate on what part is unclear or insufficiently detailed and I will take a look at improving them.

**Comment by linonetwo** [comment]
I think step 1's "Alternatively," is unclear, I know it will take words to describe the instruction, but I think it worth.

The problem with drag&drop is it won't save plugin to the `plugins/` folder for nodejs wiki (for now).

**Comment by sobjornstad** [comment]
> I think step 1's "Alternatively," is unclear, I know it will take words to describe the instruction, but I think it worth.

I don't really think that explaining how to install plugins on Node.js belongs in TiddlyRemember's documentation. That is the same for any TiddlyWiki plugin. I wouldn't explain in detail how to double-click on an installer in my downloads folder if this were a Windows program either, because that's not something my application controls; I would just say “run the installer” and then provide any instructions needed after that.

However, linking to the TiddlyWiki documentation on the structure of Node.js plugin folders makes good sense here. I've changed this section to read (with appropriate links):

> First you need to install the TiddlyWiki plugin into your wiki. To do so, drag and drop this link into your TiddlyWiki: TiddlyRemember.
>
> Alternatively, if you're running on Node.js, you might prefer to (but do not have to) install the plugin as a plugin folder instead. See Installing custom plugins on Node.js in the TiddlyWiki documentation for details on the available options. You can find TiddlyRemember's code in this GitHub repository.

-----

> The problem with drag&drop is it won't save plugin to the plugins/ folder for nodejs wiki (for now).

Aside from making it a little harder to update, AFAIK it doesn't really matter if it's there or not. I do prefer installing into the plugins folder myself, but either way works fine in practice.

I actually don't understand why, because according to the docs plugins installed via drag-and-drop are not usable when running directly under Node rather than a browser. But I've tested it, and syncing TiddlyRemember (which requires tiddlers in the TiddlyRemember plugin to work!) still works fine when you install TiddlyRemember via drag-and-drop. Maybe that caveat only applies to JavaScript modules or something.
